models/config.py

```diff
@@ -9,12 +8,23 @@ class Config(object):
 	def __init__(self):
 		self.base_request = BaseRequest()
 		self.settings = Settings()
+		self._config = None
+
+	def _get_config(self, key):
+		if self._config:
+			if key in self._config:
```

Comment:
This check is not needed. If `_config` is defined the `get_all()` method will do nothing, so the `key` won't appear there anyway